tests/EnqueueMessageProcessorTest.php

```diff
@@ -179,31 +179,31 @@ public function testShouldProxyQueryToQueryBusAndReturnReply()
     /**
      * @return \PHPUnit_Framework_MockObject_MockObject|CommandBus
```

Comment:
I decided to keep it because of `PHPUnit_Framework_MockObject_MockObject` so Phpstorm does not complain on mock methods